comment_on_staging_test_change.sh

```diff
@@ -0,0 +1,26 @@
+#! /bin/sh
+
+# Run the script only in case if it is a PR
+if [ ! -z "$TRAVIS_PULL_REQUEST" ]; then
```

Comment:
I suggest to use an early return instead, e.g. `if [ -z "$TRAVIS_PULL_REQUEST" ] && exit 1`